docs/changelogs/v2.3.1.md

```diff
@@ -0,0 +1,46 @@
+Here's the changelog with improved formatting and clarity. The changes focus on readability and structure.
+
+---
+
+## Changelog
+
+### Important Features
+
+#### New Builtin Icons Page
+
+- **URL**: `CODER_ACCESS_URL/icons` e.g., `https://coder.mycompany.com/icons`
+  ![Builtin icons page](../images/icons-gallery.png)
+
+#### Icon Selector in Emoji Picker
+
+![Icon selector](https://user-images.githubusercontent.com/418348/273346591-53837fd8-f3ce-4a71-b73e-bda9a62bb7c3.png)
+
+**Impact**: Both features improve usability in finding and using builtin icons.
+**Docs**: [Icons Documentation](https://coder.com/docs/v2/latest/templates/icons)
```

Comment:
I mentioned this in the [previous changelog](https://github.com/coder/coder/releases/tag/v2.3.0). Any idea why the hash showed up twice?